charts/customer-center/templates/ingress.yaml

```
@@ -0,0 +1,39 @@
+{{- if .Values.ingress.enabled -}}
+{{- $backendServiceName := .Values.backend.service.name }}
+{{- $backendExternalPort := .Values.backend.service.externalPort }}
+{{- $frontendServiceName := .Values.frontend.service.name }}
```

```suggestion
{{- $frontendServiceName := nclude "customerCenter.fullname" . }}-frontend
```
not sure if this works tho.